com.unity.render-pipelines.high-definition/Documentation~/Terrain-Lit-Shader.md

```diff
@@ -40,6 +40,7 @@ If you use the **Paint Holes** tool on your terrain, make sure to enable the **T
 
 | **Property**                  | **Description**                                              |
 | ----------------------------- | ------------------------------------------------------------ |
+| **Use Opacity as Density Blend** | Specifies whether HDRP should render the Terrain Layer using the value stored in the alpha channel of the Terrain Layer's **Diffuse Map** Texture instead of the usual splatmap weight or height value from the **Mask Map**. The alpha channel value is used as a threshold value for the layer blending. If the splat weight value combined with the alpha channel value is greater than one, the Terrain Layer will be visible as part of the blended result. After this calculation, the modified weights are then normalized in the shader. This option becomes available on each Terrain Layer when the Terrain's TerrainLit Material has **Enable Height-based Blend** disabled. |
```

Comment:
Indicates whether HDRP uses the alpha channel of the **Diffuse Map** as a threshold value for layer blending. If the splatmap's weight value plus the alpha channel value is greater than one, the Terrain Layer is visible as part of the blended result. After this calculation, the shader normalizes the modified weights.
<br/>
This property only appears when you enable **Enable Height-based Blend** on the Terrain Layer's TerrainLit Material.

Reply:
^^ This is my initial thought. One thing that I want to know is the significance of the shader normalizing the modified weights. What are they used for?

Reply:
Reads well to me. Technically speaking, the shader normalizes the weights after the density blend is calculated so that at least one layer is always visible and it tends to be the layer with the greatest authored splatmap weight but maybe that's not necessary to know?

Reply:
going to remove this from the doc. sharlene is adding this info to the Terrain Layer manual page https://docs.unity3d.com/Manual/class-TerrainLayer.html